src/renderer/renderElement.ts

```diff
@@ -836,6 +836,18 @@ export const renderElementToSvg = (
   const cy = (y2 - y1) / 2 - (element.y - y1);
   const degree = (180 * element.angle) / Math.PI;
   const generator = rsvg.generator;
+
+  // element to append node to, most of the time svgRoot
+  let root = svgRoot;
+
+  // if the element has a link, create an anchor tag and make that the new root
+  if (element.link) {
+    const anchorTag = svgRoot.ownerDocument!.createElementNS(SVG_NS, "a");
+    anchorTag.setAttribute("href", element.link);
+    root.appendChild(anchorTag);
+    root = anchorTag;
```

Comment:
I'm not a huge fan of this `root` variable, but I wasn't totally sure of how to have this work for every kind of element generically, so making a generic root and updating it seemed like the best approach, I'm open to any suggestions though!